models/candidate.rb

```diff
@@ -1,6 +1,7 @@
 class Candidate < ActiveRecord::Base
   include HasCalculations
 
+  has_one :committee, foreign_key: 'Filer_ID', primary_key: 'FPPC'
```

Comment:
Curious about what prompted this change.

Reply:
This allows us to do `candidate.committee` to get the committee associated with the candidate, rather than doing the query explicitly. This is beneficial because it can be loaded in bulk by ActiveRecord, which is a (very minor) performance improvement.

https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations

Reply:
It is unrelated to the search indexing, except that when I was cleaning up the duplicated contributor list indexing, I noticed the potential for this.